components/CookiesPopup/CookiesPopup.tsx

```diff
@@ -0,0 +1,22 @@
+import { useLocalStorage } from '../../hooks/useLocalStorage';
+
+import styles from './cookiesPopup.module.scss';
+
+export const CookiesPopup = () => {
+  const [accepted, setAccepted] = useLocalStorage('cookies-accepted', '');
+
+  if (!accepted && accepted !== null) {
```

Comment:
Jaka logika stoi za tym ifem?

Reply:
`null` zwracany jest, gdy dane z localStorage nie zostały jeszcze załadowane. Dodanie tego warunku ma na celu zapobiec flashowi tego popupu na ułamek sekundy, zanim dane z localStorage trafią do stanu

Reply:
To nie lepiej byłoby, żeby domyślnie `accepted` przyjmowało jakąś wartość?

Reply:
Tak